docs/learn/learn-nominator.md

```diff
@@ -212,9 +222,9 @@ fully distributed to one or more validators. That being said, you may not receiv
 nominated very few validator candidates and no one got elected, or your stake is small and you only
 selected oversubscribed validators, or the validator you are nominating has 100% commission. It is
 generally wise to choose as many trustworthy validators as you can (up to
-{{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominations" defaultValue={16}/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :kusama }}) 
-to reduce the risk of none of your nominated validators being
-elected.
+{{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominations" defaultValue={16}/>) :polkadot }}
+{{ kusama: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/>) :kusama }}
```

Comment:
I pushed a small change here.  I noticed you left both components on the same line because you were getting an added space by the parentheses.  This can be fixed by adding the closing `)` inside the conditional rendering braces for both.